content/en/docs/Getting started/_index.md

```diff
@@ -5,23 +5,122 @@ weight: 100
 description: Quick Baker setup instructions
 ---
 
-Baker is a Go library and it must be included into your Go program.
+Do you think Baker suits your needs? Good! Now let's see together how to start using it.
 
-To create a new Go project using Go modules and adding Baker:
+Baker is a Go library and, to be used, it needs to be included into a Go program.  
+In this page we describe the simplest way to use Baker. At the end of it, we recommend reading the
+[Baker Core Concepts](/docs/core-concepts/) and then have a deep dive in the
+[How-to pages](/docs/how-tos/).
+
+## Add Baker...
+
+### ...to a brand-new project
+
+To create a new Go project (using Go modules) and add Baker, these are the suggested steps:
 
 ```sh
 mkdir myProject
 cd myProject
 go mod init github.com/myUser/myProject
-go get github.com/adroll/baker
+go get github.com/AdRoll/baker
 ```
 
+### ...to an existing project
+
 If you are adding Baker to a project already configured to use Go modules, just type:
 
 ```sh
 cd myProject
-go get github.com/adroll/baker
+go get github.com/AdRoll/baker
+```
+
+## Use Baker
+
+Once Baker has been added to the project, let's see how to use it, with a minimalistic example.
+
+This code comes from the [cli](https://github.com/AdRoll/baker/tree/main/examples/cli)
+example. You can find more examples in the
+[`examples/` folder]((https://github.com/AdRoll/baker/tree/main/examples/cli)) of the project.
+
+```go
+package main
+
+import (
+    "log"
+
+    "github.com/AdRoll/baker"
+    "github.com/AdRoll/baker/filter"
+    "github.com/AdRoll/baker/input"
+    "github.com/AdRoll/baker/output"
+    "github.com/AdRoll/baker/upload"
+)
+
+func main() {
+    // Add all available components
+    comp := baker.Components{
+        Inputs:  input.All,
+        Filters: filter.All,
+        Outputs: output.All,
+        Uploads: upload.All,
+    }
+
+    // run Baker
+    if err := baker.MainCLI(comp); err != nil {
+        log.Fatal(err)
+    }
+}
+```
+
+To create the binary, just build it:
+
+```sh
+cd myProject
+go build -o baker-example .
 ```
 
-Read [Baker Core concepts](/docs/core-concepts/) to know how Baker works or go to the
+### Configuration
+
+In the example above we use [`baker.MainCLI`](https://pkg.go.dev/github.com/AdRoll/baker#MainCLI),
+an utility function that hides a lot of commonly used setup and requires a
+[TOML file](https://github.com/toml-lang/toml) as first command line parameter.
+
+The TOML file should the Baker configuration that suits your needs.  
+For details about how to configure Baker,
+[read the dedicated page](http://localhost:1313/docs/how-tos/pipeline_configuration/).
+
+A simple example, still coming from the [cli](https://github.com/AdRoll/baker/tree/main/examples/cli)
+example, is the following:
+
+```toml
+[fields]
+names = ["timestamp", "source", "target"]
+
+[input]
+name = "List"
+
+	[input.config]
+	files=["testdata/input.csv.zst"]
```

Comment:
Up to this point a lambda user might have followed but now know what a zstd file is, nor have zstandard installed. We should probably use a more common file type, and rather than creating a small pipeline that just recompress in another format, maybe have one with a simple filter that changes and/or removes some fields.
Also it would be nice if we added a stdin input (reading uncompressed log lines), to get compression out of the way for tutorials, etc.

Reply:
Ok, I'm changing the example and get the one from the basic example, with a filter. Also, I changed it a bit to use gz.
I like the idea of using a toml from an example (that is linked at the top) so that the user can jump to it